deluge/core/core.py

```diff
@@ -101,7 +101,7 @@
 
 
 class Core(component.Component):
-    def __init__(self, listen_interface=None, read_only_config_keys=None):
+    def __init__(self, listen_interface=None, outgoing_interfaces=None, read_only_config_keys=None):
```

Comment:
I don't know how long it has been like this but it is multiple just like listen_interfaces/listen_interface thats kinda why i was a little confused as I was seeing a lot of "listen_interface" and not "listen_interfaces" when I was looking how that was implemented. From the online documents for libtorrent "a comma-separated list of IP port-pairs.". However at least more recently I believe this can be a interface eg. eth0,wlan0... On linux or unix type OSs. However in deluge you have it set to verify against a single ip and no interface so a rework would be needed to support those.

Reply:
so something like https://github.com/deluge-torrent/deluge/pull/180/commits/438b2fa40b34bc3e357f6e3cf94b866c257b24d9 is what I was thinking however this still doesn't cover network interfaces but does fix checking ips that may or may not have more than one or an ipv6 address with "[]" which we also have to separate out to get the ip separate from the port. If that made sense...lol

Reply:
Can we concentrate on just a single outgoing interface for now? We can add code to support multiple interfaces and interface name in future PRs. 

Keeping a PR simple will help it get merged sooner. 

Also can you add a changelog entry

Reply:
sure. I was a bit confused before I was like how is this setting getting applied if the libtorrent setting is "listen_interfaces" but every where in deluge its 'listen_interface" but i see it now deluge\core\preferencesmanager.py>__set_list_on has apply_session_settings "listen_interfaces". *facepalm*